docs/examples-setup.md

```diff
@@ -2,17 +2,24 @@
 
 ## iOS
 
-1. Install dependencies and open the workspace:
+1. If you don't have bundler gem installed:
+
+```
+gem install bundler
+```
+
+2. Install dependencies and open the workspace:
 
 ```
 cd example
 npm install
 cd ios
-pod install
+bundle install
+bundle exec pod install
```

Comment:
oh are you saying that `bundle exec` runs `bundle install` automatically? I don't know much about bundler tbh